tests/Doctrine/ODM/MongoDB/Tests/Aggregation/AggregationOperatorsProviderTrait.php

```diff
@@ -194,6 +194,11 @@ public static function provideExpressionOperators()
                 'operator' => 'exp',
                 'args' => ['$field'],
             ],
+            'exprOp' => [
+                'expected' => ['$expr' => ['$eq' => ['$field', '$otherField']]],
+                'operator' => 'exprOp',
+                'args' => [['$eq' => ['$field', '$otherField']]],
+            ],
```

Comment:
Suggestion to account for the new method names:

```suggestion
            'aggregationExpression' => [
                'expected' => ['$expr' => ['$eq' => ['$field', '$otherField']]],
                'operator' => 'aggregationExpression',
                'args' => [['$eq' => ['$field', '$otherField']]],
            ],
```

In the long run, we may want to also pass expression objects to these operator methods, but that's something we should incorporate better than creating separate items in the data provider. Just making a mental note in case anyone wants to take a crack in a separate PR.